memegen/domain/image.py

```diff
@@ -92,7 +92,7 @@ def make_meme(top, bottom, font, background, path, match_font_size=False):
                         bottom, bottom_font, bottom_font_size)
 
     log.info("Generating image: %s", path)
-    return img.save(path, format=img.format)
+    return img.save(path, format='png')
```

Comment:
yes... what if we check before we force anything, and try to keep format if possible